dependency-check-suppressions.xml

```diff
@@ -34,4 +34,11 @@
         <cve>CVE-2016-5394</cve>
         <cve>CVE-2016-6798</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
```

Comment:
which package is this currently affecting? Should it have an until date? (appreciate its false positive though good to have as few supressions as possible)

Reply:
By mistake I have added this suppress, I have remove it now